python/ray/data/datasource/parquet_datasource.py

```diff
@@ -83,7 +83,7 @@
 PARQUET_ENCODING_RATIO_ESTIMATE_DEFAULT = 5
 
 # The lower bound size to estimate Parquet encoding ratio.
-PARQUET_ENCODING_RATIO_ESTIMATE_LOWER_BOUND = 2
+PARQUET_ENCODING_RATIO_ESTIMATE_LOWER_BOUND = 1
```

Comment:
Can you also add a comment that the lower bound set to 1 to avoid sampling giving wrong estimation that estimated size to be smaller than on-disk size.

Reply:
@c21 are we sure that's always true? The ratio isn't against the on-disk size, but rather the "byte size of all the uncompressed column data."

Reply:
Sorry, after a second look, you are right. Thanks.